README.md

```diff
@@ -295,6 +295,11 @@ from which I leared that it had been previously invented a decade earlier.
 //#define TEST_H "qtl/module.h"
 #include TEST_H
 ```
+### test.bash
+```bash
+# compile and run demo
```

Comment:
Users can  type SQLish insert and select statements here.
If there is something you don't understand, please ask questions.
We can then try to either adjust your expectations by explaining what it is doing,
or try to adjust what it is doing to better coincide with your expectations.